packages/xo-server-sdn-controller/src/ovsdb-client.js

```diff
@@ -1,15 +1,25 @@
 import assert from 'assert'
 import createLogger from '@xen-orchestra/log'
-import forOwn from 'lodash/forOwn'
 import fromEvent from 'promise-toolbox/fromEvent'
 import { connect } from 'tls'
+import { forOwn, toPairs } from 'lodash'
 
 const log = createLogger('xo:xo-server:sdn-controller:ovsdb-client')
 
 const OVSDB_PORT = 6640
 
 // =============================================================================
 
+function toOvsdbMap(object) {
+  return ['map', toPairs(object)]
+}
+
+function toOvsdbSet(object) {
+  return ['set', toPairs(object)]
+}
```

Comment:
Is `Ovsdb` term necessary here?

Reply:
I guess not.
Is it a standard set/map representation?